test/hooks/achievement-rule-checker.test.js

```diff
@@ -144,11 +150,12 @@ describe('\'achievement-rule-checker\' hook', () => {
       }
     });
 
-    assert.deepEqual(result, []);
+    assert.lengthOf(result, 1);
+    assert.deepInclude(result[0], {name: achievement_name, current_amount: 0, total_amount: 2});
   });
 
 
-  it('gives achievement maxAwarded times', async () => {
+  it('gives achievement maxAwardedTotal times', async () => {
```

Comment:
Now there is no test for `maxAwarded` anymore although we should probably test that as well. Maybe even combined with `maxAwardedTotal`.

Reply:
You're right, I was a bit lazy there. I've now tested both `maxAwarded` and `maxAwardedTotal` using a `for`-loop.